crypto/engine/eng_cryptodev.c

```diff
@@ -594,9 +609,13 @@ static const EVP_CIPHER *cryptodev_des_cbc(void)
             || !EVP_CIPHER_meth_set_init(cipher, cryptodev_init_key)
             || !EVP_CIPHER_meth_set_do_cipher(cipher, cryptodev_cipher)
             || !EVP_CIPHER_meth_set_cleanup(cipher, cryptodev_cleanup)
-            || !EVP_CIPHER_meth_set_impl_ctx_size(cipher, sizeof(struct dev_crypto_state))
-            || !EVP_CIPHER_meth_set_set_asn1_params(cipher, EVP_CIPHER_set_asn1_iv)
-            || !EVP_CIPHER_meth_set_get_asn1_params(cipher, EVP_CIPHER_get_asn1_iv)) {
+            || !EVP_CIPHER_meth_set_impl_ctx_size(cipher,
+                                                  sizeof(struct
```

Comment:
In most cases we use sizeof on the actual variable, or ```sizeof(*pointer)```  Nothing to change for those.  But here, cipher is of type EVP_CIPHER, and the sizeof is of something else.  I consider that a "magic number."  Not everyone necessarily agrees, I'm sure, but I hope the reasoning at least makes sense.